### PR TITLE
Adds avatar upload/removal functionality to Basic Information

### PR DESCRIPTION
**Pull request by sizzzzle**
## Description
- Exports `isValid` from `_form`
- Adds a new Upload Avatar component
- Binds Upload Avatar in Basic Information

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: apps/courses/src/components/UploadAvatar.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      )
+      .then(() =>
+        toast({
+          ...toastConfig,
+          title: 'Profile photo uploaded',
+          description: 'Please refresh to see this change live...',
+          status: 'success',
+        })
+      )
+      .catch(({ message }) =>
```
Comment (cereallarceny) [inline apps/courses/src/components/UploadAvatar.tsx]: Use handleErrors

File: apps/courses/src/components/forms/_form.tsx [hunk 2]
```diff
@@ -315,7 +319,7 @@ export default ({
           </Button>
         )}
         {typeof submit !== 'string' &&
-          submit(!isDirty || (isDirty && !isValid), isSubmitting)}
+          submit(!isDirty || (isDirty && !isValid), isSubmitting, isValid)}
```
Comment (cereallarceny) [inline apps/courses/src/components/forms/_form.tsx]: Remove isValid

File: apps/courses/src/components/forms/_form.tsx [hunk 3]
```diff
@@ -49,7 +49,11 @@ interface FormProps {
   onSubmit: (any) => void;
   fields: (Field | Field[])[];
   submit?:
-    | ((isDisabled: boolean, isSubmitting: boolean) => React.ReactNode)
+    | ((
+        isDisabled: boolean,
+        isSubmitting: boolean,
+        isValid: boolean
```
Comment (cereallarceny) [inline apps/courses/src/components/forms/_form.tsx]: Remove isValid

File: apps/courses/src/components/forms/users/BasicInformation.tsx [hunk 4]
```diff
@@ -43,13 +51,43 @@ export default ({
...
   const user: firebase.User = useUser();
+  const storage = useStorage();
   const auth = useAuth();
   const db = useFirestore();
   const toast = useToast();
 
   const dbUserRef = db.collection('users').doc(user.uid);
   const dbUser: User = useFirestoreDocData(dbUserRef);
 
+  const removeAvatar = () => {
```
Comment (cereallarceny) [inline apps/courses/src/components/forms/users/BasicInformation.tsx]: Move this to component

File: apps/courses/src/components/forms/users/BasicInformation.tsx [hunk 5]
```diff
@@ -140,16 +177,34 @@ export default ({
...
-      schema={schema}
-      fields={fields}
-      submit="Save Changes"
-      isBreathable
-    />
+    <>
+      <Box mb={8}>
+        <Flex direction="row">
+          <Text mb={2} fontWeight={700} fontSize="sm" color="gray.700" mr={4}>
+            Change your profile picture
```
Comment (cereallarceny) [inline apps/courses/src/components/forms/users/BasicInformation.tsx]: Put all of this in the component but take a prop for the label

**Review by cereallarceny — CHANGES_REQUESTED** [review]

File: apps/courses/src/components/UploadAvatar.tsx [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+    Resizer.imageFileResizer(
+      file,
+      400,
+      400,
+      'png',
+      100,
+      0,
+      (blob) => {
+        // @ts-ignore
+        const resizedImage = new File([blob], user.uid, {
```
Comment (sizzzzle) [inline apps/courses/src/components/UploadAvatar.tsx:60]: @cereallarceny it was complaining about blob :/

**Review by cereallarceny — APPROVED** [review]

**Pull request merged**